src/Npgsql/NpgsqlDataSourceBuilder.cs

```diff
@@ -295,6 +295,17 @@ public NpgsqlDataSourceBuilder UseRootCertificate(X509Certificate2? rootCertific
         return this;
     }
 
+    /// <summary>
+    /// Sets the <see cref="X509Certificate2Collection" /> that will be used validate SSL certificate, received from the server.
+    /// </summary>
+    /// <param name="rootCertificates">The CA certificates.</param>
+    /// <returns>The same builder instance so that multiple calls can be chained.</returns>
+    public NpgsqlDataSourceBuilder UseRootCertificate(X509Certificate2Collection? rootCertificates)
```

Comment:
Seems like we should name the overloads that accept a collection UseRootCertificates (plural) (same for UseRootCertificatesCallback)